.circleci/src/workflows/delivery.yml

```diff
@@ -1,17 +1,14 @@
-triggers:
```

Comment:
Scheduled workflows will be phased out. Instead, it is now possible to add triggers in the circleci project settings: https://app.circleci.com/settings/project/github/digitalfabrik/integreat-app/triggers

See https://circleci.com/docs/2.0/workflows/#scheduling-a-workflow and https://circleci.com/docs/2.0/scheduled-pipelines